src/system.ts

```diff
@@ -66,21 +68,49 @@ process.on('uncaughtException', (error) => {
 
 const arweave = Arweave.init({});
 
+// Stores
+
+const txStore = new KvTransactionStore({
+  log,
+  kvBufferStore: (() => {
+    switch (config.CHAIN_CACHE_TYPE) {
+      case 'lmdb': {
+        return new LmdbKVStore({
+          log,
+          lmdbOptions: {
+            path: 'data/lmdb/partial-txs',
+            // TODO: set sensible default options for LMDB client
```

Comment:
deciding what is sensible here: https://github.com/kriszyp/lmdb-js#db-options

Reply:
testing compression - enabled [ac03e5f](https://github.com/ar-io/ar-io-node/pull/60/commits/ac03e5f40aaa2757991ac27e96772e9e9fd51166)

Reply:
through 750 blocks (staring at 800k) - minimal, but some difference,
non-compressed: 304MiB
compressed: 300MiB
```
❯ du -sh data-non-compressed.mdb
304M	data-non-compressed.mdb
❯ du -sh data-compressed.mdb
300M	data-compressed.mdb
```